### DISCUSSION TITLE
Team mention from bot not working

### DISCUSSION THREAD

**Discussion by awendt** [community/community#25608] [Actions]
<p>Hi! :wave:</p>
<p>I’m trying to mention my team from <a href="https://github.com/actions/github-script" rel="noopener nofollow ugc">actions/github-script</a> – individual mentions work alright, but team mentions are shown as plain text when sent from <code>github-actions[bot]</code>.</p>
<p>If I query the <code>body</code> attribute of an issue comment typed manually (this is successfully turned into a team mention) and compare it with the <code>body</code> attribute of an issue comment sent by <code>github-actions[bot]</code>, I can’t spot a formatting difference.</p>
<p>What’s going on here?</p>
<p>Thanks,<br>
André</p>

**Comment by nbaztec** [comment]
<p>I’m having the same problem here and not sure how to go about it. :frowning:</p>

**Comment by iCynara** [comment]
<p>I also have the same issue.  it is believed a github bug, and they have not fixed it yet.<br>
do you guys find any workaround can share?</p>

**Comment by iCynara** [comment]
<p>Hi after a few more testing, i managed to make it work.<br>
The trick is you need to give the <code>member</code> permission for you app.<br>
It is under app settings → permissions &amp; events → Organization → member → read only</p>
<p>Then i can mention <code>@ team</code> no problem</p>

**Comment by ciaransweet** [comment]
<p>Hey <a class="mention" href="/icynara">@iCynara</a> was this using an actual GitHub App or just an Action?</p>
<p>I’m hitting the same issue but with the following workflow:</p>
<pre><code class="lang-auto">name: Notify for new PR

on:
  pull_request:
    types: [opened]

jobs:
  notify-maintainers:
    runs-on: ubuntu-20.04
    steps:
      - name: Notify @chaostoolkit/maintainers
        uses: JoseThen/comment-pr@v1.1.0
        with:
          comment: |
            Hey @chaostoolkit/maintainers
            Someone just raised this PR! :construction_worker:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
</code></pre>
<p>I get the message but it’s not linked:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/4/54f1d90583fb6bdbad4e9c27fa78d2478bec4460.png" data-download-href="/uploads/short-url/c7sgmtA0bQ573NGLfLGkrEYoycM.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/9111975/181092180-433e94d7-6bb0-4221-b564-b2c2df214d5b.png" alt="image" data-base62-sha1="c7sgmtA0bQ573NGLfLGkrEYoycM" width="690" height="239" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/4/54f1d90583fb6bdbad4e9c27fa78d2478bec4460_2_690x239.png, https://aws1.discourse-cdn.com/github/original/3X/5/4/54f1d90583fb6bdbad4e9c27fa78d2478bec4460.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/4/54f1d90583fb6bdbad4e9c27fa78d2478bec4460.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/4/54f1d90583fb6bdbad4e9c27fa78d2478bec4460_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">952×330 26 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>However if I ‘edit’ the post and go to preview, it’s then linked :frowning:</p>

**Comment by moui72** [comment]
<p>I’m also having this issue via actions :frowning:</p>

**Comment by wechuli** [comment, marked as answer]
<p>The issue here is that the default <code>GITHUB_TOKEN</code> does not have permission to read teams in an organization:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/15605874/181092182-c7a7ac9e-59a9-4b3e-bb3a-4673ec5207b4.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token" target="_blank" rel="noopener nofollow ugc">Authentication in a workflow - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Its access is limited to the repository where the workflow is running. You’ll need to create a PAT for that functionality to work. If you are using <code>actions/github-script</code>, you can pass the PAT as an input (it default to the <code>GITHUB_TOKEN</code> if no other token is passed):</p>
<aside class="onebox githubblob" data-onebox-src="https://github.com/actions/github-script/blob/5d31eecbde4f80ba181f36dbc095950302fca32c/action.yml#L11">
  <header class="source">

      <a href="https://github.com/actions/github-script/blob/5d31eecbde4f80ba181f36dbc095950302fca32c/action.yml#L11" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/actions/github-script/blob/5d31eecbde4f80ba181f36dbc095950302fca32c/action.yml#L11" target="_blank" rel="noopener nofollow ugc">actions/github-script/blob/5d31eecbde4f80ba181f36dbc095950302fca32c/action.yml#L11</a></h4>



  <pre class="onebox">    <code class="lang-yml">
      <ol class="start lines" start="1" style="counter-reset: li-counter 0 ;">
          <li>name: GitHub Script</li>
          <li>author: GitHub</li>
          <li>description: Run simple scripts using the GitHub client</li>
          <li>branding:</li>
          <li>  color: blue</li>
          <li>  icon: code</li>
          <li>inputs:</li>
          <li>  script:</li>
          <li>    description: The script to run</li>
          <li>    required: true</li>
          <li class="selected">  github-token:</li>
          <li>    description: The GitHub token used to create an authenticated client</li>
          <li>    default: ${{ github.token }}</li>
          <li>    required: false</li>
          <li>  debug:</li>
          <li>    description: Whether to tell the GitHub client to log details of its requests</li>
          <li>    default: false</li>
          <li>  user-agent:</li>
          <li>    description: An optional user-agent string</li>
          <li>    default: actions/github-script</li>
          <li>  previews:</li>
      </ol>
    </code>
  </pre>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
